src/views/Pipeline.vue

```diff
@@ -117,6 +117,48 @@ export default defineComponent({
         event.target.complete();
       })
     },
+    async skipJob (job: any) {
+      const alert = await alertController
+        .create({
+          header: this.$t('Skip job'),
+          message: this.$t('Skipping will run this job at the next occurance based on the temporal expression.'),
+          buttons: [
+            {
+              text: this.$t("Don't skip"),
+              role: 'cancel',
+            },
+            {
+              text: this.$t('Skip'),
+              handler: async () => {
+                let skipTime = {};
+                const integer1 = this.temporalExpr(job.tempExprId).integer1;
+                const integer2 = this.temporalExpr(job.tempExprId).integer2
+                if(integer1 === 12) {
+                  skipTime = { minutes: integer2 }
+                }
+                else if (integer1 === 10) {
+                  skipTime = { hours: integer2 }
+                }
+                else if (integer1 === 5) {
+                  skipTime = { days: integer2 }
+                }
```

Comment:
Handle case if none of the above conditions satisfy

Reply:
Handled case if none of the above conditions satisfy.